src/migration/ember-addon/steps/move-project-root-files.ts

```diff
@@ -43,13 +43,10 @@ function moveToAddonAndTestApp(options: Options): void {
     '.template-lintrc.cjs',
     '.template-lintrc.js',
     'package.json',
-  ]);
-
-  if (packages.addon.hasTypeScript) {
-    files.add('tsconfig.json');
-  }
+    'tsconfig.json',
```

Comment:
Because of `findFiles()`, `filePaths` will include `tsconfig.json` only if the file exists. As a result, the codemod doesn't need to conditionally add `tsconfig.json`.